### DISCUSSION TITLE
How does the fluentd log driver work?

### DISCUSSION THREAD

**Discussion by nik736** [containerd/nerdctl#1525] [Q&A]
Hi there,

When using docker run I can use the following to receive docker logs to the fluentd endpoint.
`--log-driver=fluentd --log-opt=fluentd-address=remote_ip:24224`

My fluentd server is not on the same machine so I am sending it to a remote ip or hostname. When using the same with nerdctl it is not working, no logs are received and I am not sure how to debug. I am using nerdctl 1.0.0

Looking through the docs I can see the following line: `The fluentd daemon must be running on the host machine.` But this does not make sense for me, as I am running fluentd on another machine.

How am I able to get this up and running?

Any help is appreciated, thanks!

**Comment by fahedouch** [comment]
Hi @nik736 ,

Did you get any output when running with `nerdctl --debug run`  mode ? 
Where did you find this line `The fluentd daemon must be running on the host machine.` ? 

Can you please share your `fluentd` configuration ? as we are using a [fluent-client-library](github.com/fluent/fluent-logger-golang/fluent) to push log entries, I believe that you need the [forward protocol plugin](https://docs.fluentd.org/input/forward) configured on the fluentd side
